unitTests/ui/LiveAnalyticsClientTest.ts

```diff
@@ -428,6 +428,30 @@ export function LiveAnalyticsClientTest() {
       );
     });
 
+    it(`when specifying a result having a #permanentid as an argument for #logCustomEvent,
```

Comment:
We should have a test that check that if it is not present, then nothing changes compared to the previous version and the metadata does not contain "empty" or null values.